go.mod

```diff
@@ -79,6 +79,13 @@ require (
 require (
 	github.com/IGLOU-EU/go-wildcard v1.0.3
 	github.com/sigstore/k8s-manifest-sigstore v0.3.1-0.20220810053329-14f7cab4fd52
+	github.com/smallstep/zcrypto v0.0.0-20210924233136-66c2600f6e71
```

Comment:
@zeborg can we remove this dependency and use golang stdlib's x509 library https://pkg.go.dev/crypto/x509?

Reply:
The issue had mentioned a reference to Step CLI, so I was under the impression that we had to keep it in line with its implementation. But sure, I'll just replace it with the standard library instead.